src/utils/AccessibleButton.ts

```diff
@@ -113,23 +113,13 @@ export class AccessibleButton {
   }
 
   private ensureDifferentiationBetweenKeyboardAndMouseFocus() {
-    const classOnPress = 'coveo-accessible-button-pressed';
-    const classOnFocus = 'coveo-accessible-button-focused';
+    const classWhenPressed = 'coveo-accessible-button-pressed';
```

Comment:
🤔 

It does exactly the same thing as before for all selectable components when you tab through them versus click... ?

Reply:
This bundled with the css `:active` state does it from what I've tried